ci/Dockerfile

```diff
@@ -89,6 +89,14 @@ RUN GOLANG_SHA256="e330e5d977bf4f3bdc157bc46cf41afa5b13d66c914e12fd6b694ccda65fc
  && rm "${GOLANG_TARBALL}" \
  && sudo ln -s /usr/local/go/bin/go /usr/local/bin
 
+# Install Protobuf
+RUN PROTOC_SHA256="b9415563e9e147b95f5d3573c92f9e25117a1ad0123cd0b6bd5e8590ea26bd7d" \
+ && PROTOC_TARBALL="protoc-gen-go.v1.35.1.linux.amd64.tar.gz" \
```

Comment:
I think this version is too new for `status-go` 
we could try using `v1.34.1` -> https://github.com/status-im/status-go/blob/55bad8fd7b48439ded3abef549c11509a4388272/go.mod#L71